lib/cache/cache.go

```diff
@@ -202,11 +202,14 @@ func ForOldRemoteProxy(cfg Config) Config {
 		{Kind: types.KindAuthServer},
 		{Kind: types.KindReverseTunnel},
 		{Kind: types.KindTunnelConnection},
+		{Kind: types.KindAppServer},
 		{Kind: types.KindAppServer, Version: types.V2},
+		{Kind: types.KindApp},
 		{Kind: types.KindRemoteCluster},
 		{Kind: types.KindKubeService},
 		{Kind: types.KindDatabaseServer},
-		{Kind: types.KindKubeServer},
+		{Kind: types.KindDatabase},
+		{Kind: types.KindInstaller},
```

Comment:
Is this just taken from the v10 `ForRemoteProxy`? If so, it might be worth mentioning it in a comment (maybe in the godoc for `ForOldRemoteProxy`).